Playbooks/Enrich-AzureResourceGraph/azuredeploy.json

```diff
@@ -0,0 +1,420 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "metadata": {
+        "title": "Query Azure Resource Graph with HTTP input and output",
+        "description": "This playbook will query Azure Resource Graph with HTTP input and output.",
```

Comment:
From the description, I'm not sure what this playbook is doing. Can you make it more comprehensible?

Reply:
I'm unclear how to reword differently.
This is to make query to Azure Resource Graph - https://learn.microsoft.com/en-us/azure/governance/resource-graph/overview

Reply:
Something like this,
"This playbook queries Azure Resource Graph and returns azure information related to the resource like Subscription, Resourcegroups, Tags and Management groups."